examples/Bremsstrahlung/pymodules/particleCalorimeter.py

```diff
@@ -0,0 +1,283 @@
+    existing particles as well as optionally all particles which have already
+    left the global simulation volume.
+    """
+
+    def __init__(self,
+                 species_name="ph", sim="./",
+                 period=1000, num_bins_yaw=64, num_bins_pitch=64,
+                 num_bins_energy=1024, min_energy=10, max_energy=10000,
+                 logscale=False, opening_yaw=360, opening_pitch=180,
+                 pos_yaw=0, pos_pitch=0):
+        """
+        species_name
+            name of the species for which the particle calorimeter
+            should be plotted, e.g. "ph"
+        sim
+            path to simOutput directory
+        period
+            output periodicity of the histogram
+        num_bins_yaw
+            number of bins resolving the yaw angle
+        num_bins_pitch
+            number of bins resolving the pitch angle
+        num_bins_energy
+            number of bins resolving the detected energies
+        min_energy
+            minimum detectable energy in keV
+        max_energy
+            maximum detectable energy in keV
+        logscale
+            *Boolean*, if `True` the energy bins are logarithmically spaced
+        opening_yaw
+            slit opening in x in meters
+        opening_pitch
+            slit opening in z in meters
+        pos_yaw
+            yaw coordinate of the calorimeter position in degrees
+            (default: +y-direction --> 0)
+        pos_pitch
+            pitch coordinate of the calorimeter position in degrees
+            (default: +y-direction --> 0)
+        """
+        self.species_name = species_name
+        self.sim = sim
+        self.period = period
+        self.num_bins_yaw = num_bins_yaw
+        self.num_bins_pitch = num_bins_pitch
+        self.num_bins_energy = num_bins_energy
+        self.min_energy = min_energy
+        self.max_energy = max_energy
+        self.logscale = logscale
+        self.opening_yaw = opening_yaw
+        self.opening_pitch = opening_pitch
+        self.pos_yaw = pos_yaw
+        self.pos_pitch = pos_pitch
+
+        self.attribute_name = "calorimeter"
+
+        self.energy_range = np.linspace(self.min_energy,
+                                        self.max_energy,
+                                        self.num_bins_energy)
+
+        self.calo_data = np.zeros([self.num_bins_energy,
+                                   self.num_bins_pitch,
+                                   self.num_bins_yaw], dtype=np.float64)
+        self.hist_data = np.zeros([self.num_bins_energy, ], dtype=np.float64)
+
+        self.sim_unit = None
+
+    def load_step(self,
+                  step=5000):
+        """
+        Load an HDF5 File of a certain time step
+
+        time
+            PIConGPU output time step
+        """
+        # convert the time step into a string
+        self.stepStr = str(int(step))
+
+        self.calo_file = self.sim + "/simOutput/" \
```

Comment:
please put these lines into round brackets in order to avoid the backslashes, i.e.:
```
self.calo_file = (
  self.sim + "/simOutput/" +
  self.species_name + "_" +
  self.attribute_name + "/"
)
```